### PR TITLE
Metrcis 01

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
changed based on review comments
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by andrewferlitsch** [comment]
5: --> 104 with open(file_path) as f:
Step #5:     105     return yaml.safe_load(f)
Step #5: 
Step #5: FileNotFoundError: [Errno 2] No such file or directory: 'tabular_classification_pipeline.json'

File: notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "cells": [
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>There seems to be a problem with the heading ### Datasets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>combine with above cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>looks like a problem with the heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>tabular classification_pipeline.json -&gt; tabular_classification_pipeline.json</p><p> </p><p>replace the space with underscore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>removed replace().</p><p>replace space with underscore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>same here with remove replace() and replace space with underscore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb:2]: <p>Delete only what is created in notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

**Comment by udaypunna** [comment]
New branch and PR created for this Notebbok.
Merged [#847](https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/847) into main.

**Pull request closed without merging**